entrypoint.sh

```diff
@@ -73,9 +73,9 @@ if expr match $1 '.*supervisord' >/dev/null; then
   if [ -f $aliases ]; then
     newaliases
   fi
-  
+
   # Configure authentification to relay if needed
-  if [ -n "${RELAY_LOGIN}" -a -n "${RELAY_PASSWORD}" ]; then
+  if [ \( -n "${RELAY_LOGIN}" -a -n "${RELAY_PASSWORD}" \) -o \( -f /etc/postfix/sasl_passwd \) ]; then
```

Comment:
I approuve this changes, I've never tested the sasl postfix client login using a hash database :)